test/plugin.test.ts

```diff
@@ -102,6 +119,77 @@ describe("scalaJSPlugin", () => {
       .toBeNull();
   }, testOptions);
 
+  testBothModes(it, "works with a project with subprojects", async (mode, suffix) => {
+    const [plugin, fakePluginContext] = setup({
+      subprojects: [
+        {
+          projectID: "otherProject",
+          uriPrefix: "foo",
+        },
+        {
+          projectID: null,
+          uriPrefix: "bar",
+        },
+      ]
+    });
+
+    await plugin.configResolved.call(undefined, { mode: mode });
+    await plugin.buildStart.call(fakePluginContext, {});
+
+    expect(normalizeSlashes(await plugin.resolveId.call(fakePluginContext, 'foo:main.js')))
+      .toContain('/testproject/other-project/target/scala-3.2.2/otherproject-' + suffix + '/main.js');
+    expect(normalizeSlashes(await plugin.resolveId.call(fakePluginContext, 'bar:main.js')))
+      .toContain('/testproject/target/scala-3.2.2/testproject-' + suffix + '/main.js');
+
+    expect(await plugin.resolveId.call(fakePluginContext, 'scalajs/main.js'))
+      .toBeNull();
```

Comment:
```suggestion
    expect(await plugin.resolveId.call(fakePluginContext, 'scalajs:main.js'))
      .toBeNull();
```
would make more sense, wouldn't it?

Reply:
Correct. BTW, the `scalajs/main.js` is copied from other tests. Maybe we should fix it in the other places, although it is out of scope of this PR.